### ISSUE TITLE
Separate global search options 

### ISSUE THREAD

**Issue by dillingham** [TitasGailius/nova-search-relations#7]
Nova: 1.2.9
Laravel 5.7

### Description

I find that the options I'd like to search above the table are different than those of global.

For instance, a resource may have a user relation. That relation appears on the table.
A person searching would see that user's name & expect the table search to return it 
That's what makes this package awesome!

However, if I search that user in global search, I don't want to see that resource.

### Solution

I would love `$globalSearchRelations` & `$searchRelations` separated

Non breaking: 

```php
if(isset($globalSearchRelations) == false) 
    $globalSearchRelations = $searchRelations;
```

**Comment by TitasGailius** [comment]
Implemented something like this in `1.0.4` release. https://github.com/TitasGailius/nova-search-relations/commit/1e7408a678b9258537cb15ceb58a56016501243a

**Comment by dillingham** [comment]
Sorry! Thank you!
